application-templates/starter/src/components/channel-details/channel-details.jsx

```diff
@@ -84,55 +85,74 @@ const ChannelDetails = (props) => {
   );
 
   return (
-    <ChannelsDetailsForm
-      initialValues={docToFormValues(channel, projectLanguages)}
-      onSubmit={handleSubmit}
-      isReadOnly={!canManage}
-      dataLocale={dataLocale}
-    >
-      {(formProps) => {
-        return (
-          <FormModalPage
-            title={formatLocalizedString(
-              {
-                name: formProps.values?.name,
-              },
-              {
-                key: 'name',
-                locale: dataLocale,
-                fallbackOrder: projectLanguages,
-                fallback: NO_VALUE_FALLBACK,
-              }
-            )}
-            isOpen
-            onClose={props.onClose}
-            isPrimaryButtonDisabled={
-              formProps.isSubmitting || !formProps.isDirty || !canManage
+    <>
+      <ApplicationPageTitle
+        additionalParts={[
+          `${formatLocalizedString(
+            {
+              name: transformLocalizedFieldToLocalizedString(
+                channel?.nameAllLocales ?? []
+              ),
```

Comment:
This is already done in the form values, so we don't need to do it again.

I would suggest to render the `<ApplicationPageTitle>` within the `<FormModalPage>` children and reuse the same value used for the `title` prop (you want to extract it into a variable).

See below